Dockerfile.tools

```diff
@@ -23,12 +23,18 @@ RUN pip2 install ansible pywinrm
 # Make Ansible happy with arbitrary UID/GID in OpenShift.
 RUN chmod g=u /etc/passwd /etc/group
 
+# Seems ansible-playbook doesn't work with ansible.cfg
+RUN mkdir -p /.ansible
+RUN chmod 777 -R /.ansible
+
+
 # Allow building the WMCB and creation of /go/.cache directory
 RUN chmod -R g=u+w /go
 
 # Explicitly set GOCACHE environment variable
 ENV GOCACHE="/go/.cache"
 ENV PATH="${PATH}:/usr/local/go/bin"
 ENV GOPATH="/go"
+ENV ANSIBLE_LOCAL_TEMP="/tmp"
```

Comment:
I wonder if we need to create an `ansible.cfg` file with this option and place it in the WSU playbook directory in the CI image. `ansible-playbook` will apparently look at that.